### PR TITLE
Upgrade/MC test persistence cleanup items

### PR DESCRIPTION
**Pull request by michael-cico**
# Description

Cleanup items for upgrade/MC-upgrade tests
- Fix up uninstall log dumps, update archive patterns
- rename KIND/persistence file to be more general
- Capture install and upgrade YAMLs in upgrade tests

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [x] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: ci/upgrade/Jenkinsfile [hunk 1]
```diff
@@ -208,12 +208,12 @@ pipeline {
                     }
                     post {
                         always {
-                            archiveArtifacts artifacts: "acceptance-test-operator.yaml,downloaded-operator.yaml", allowEmptyArchive: true
```
Comment (desagar) [inline ci/upgrade/Jenkinsfile:211]: do you think we don't need the operator yamls anymore? Or are those filenames not used anymore?

Comment (michael-cico) [reply ci/upgrade/Jenkinsfile:211]: probably not...in this case we apply an old release manifest directly off the web for install, so they aren't there anyway.

Comment (michael-cico) [reply ci/upgrade/Jenkinsfile:211]: actually this line was moved/replaced by 226.

Comment (michael-cico) [reply ci/upgrade/Jenkinsfile:211]: Sorry, misread the comment/question... so, the fuller, more correct answer is, we don't download those files in this test.  We do a deploy directly from a release so they don't exist on the runner.  So I removed them from the archive statement.

**Review by desagar — APPROVED** [review]

File: ci/upgrade/Jenkinsfile (hunk 1 above)
Comment (michael-cico) [reply ci/upgrade/Jenkinsfile:211]: Now the archive statement on 226 just archives the VZ CR and the install logs

**Pull request merged**